### ISSUE TITLE
[Redesign] Manage when cam/mic is not authorized 

### ISSUE THREAD

**Issue by HugoAverty** [workadventure/workadventure#4435]
![Screenshot 2024-11-27 à 16 53 43](https://github.com/user-attachments/assets/af2284c2-e2f6-4961-939a-1b69734e3ae8)

We should display something else than "Couper la caméra" / "Couper le microphone"
Handle that use case

**Comment by Moini** [comment]
Yes, please. I've just tested the new location of the camera image and it's set so you can't ever get rid of it. This is so annoying when you're just walking through the map, not even intending to meet up with anyone. 

* Camera image of self needs to be hideable completely and forever, and if possible, movable, too. I use two screens. Being able to have the camera image on one of them is useful, especially when you're screensharing.
* All that nagging about not having a camera connected needs to go. When I already said I'm not using one, and disallow it, that was my decision, and any kind of nagging is only going to make me ditch Workadventure as 'unusable'. I may have a bad hair day, or not want to let people see me. Why isn't that decision respected? Just walking in the 'headquarters' for five minutes was a whack-a-mole with that silly camera image.
